platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationManagerImpl.java

```diff
@@ -107,14 +109,10 @@ public ExportMigrationManagerImpl(
 
   @SuppressWarnings(
       "squid:S2095" /* up to the caller of this method to close it; exception is thrown if the stream cannot be created in the first place */)
```

Comment:
❓ Not sure the suppression is still warranted here

Reply:
Removed